app-text/multimarkdown/multimarkdown-6.4.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Lightweight markup processor to produce HTML, LaTeX, and more"
+DESCRIPTION="MultiMarkdown, or MMD, is a tool to help turn minimally marked-up \
+plain text into well formatted documents, including HTML, PDF (by way of LaTeX), \
+OPML, or OpenDocument (specifically, Flat OpenDocument or ‘.fodt’, which can \
+in turn be converted into RTF, Microsoft Word, or virtually any other \
+word-processing format)."
+HOMEPAGE="https://fletcherpenney.net/multimarkdown/"
+COPYRIGHT="2016-2018, Fletcher T. Penney"
+LICENSE="BSD (3-clause)
+	MIT
+	Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/fletcher/MultiMarkdown-6/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fa9daa089bc0f8bc1f69fd7365778d59210b446ce7008d03b87fb572df8ff5d3"
+SOURCE_DIR="MultiMarkdown-6-$portVersion"
+PATCHES="multimarkdown-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	multimarkdown$secondaryArchSuffix = $portVersion
+	cmd:markdown
+	cmd:mmd
+	cmd:mmd2all
+	cmd:mmd2epub
+	cmd:mmd2fodt
+	cmd:mmd2odt
+	cmd:mmd2opml
+	cmd:mmd2pdf
+	cmd:mmd2tex
+	cmd:multimarkdown
```

Comment:
I think these `cmd:`s should be versioned.

Reply:
iirc they are scripts related to the cmd:multimarkdown